src/test/scala/org/broadinstitute/dsde/workbench/sam/api/MockSamRoutes.scala

```diff
@@ -61,21 +61,24 @@ abstract class MockSamRoutes(
       oidcConfig.oauth2Routes ~
       statusRoutes ~
       oldTermsOfServiceRoutes ~
-      termsOfServiceRoutes ~
       withExecutionContext(ExecutionContext.global) {
         withSamRequestContext { samRequestContext =>
           pathPrefix("register")(oldUserRoutes(samRequestContext)) ~
             pathPrefix("api") {
               // IMPORTANT - all routes under /api must have an active user
-              withActiveUser(samRequestContext) { samUser =>
-                val samRequestContextWithUser = samRequestContext.copy(samUser = Option(samUser))
-                resourceRoutes(samUser, samRequestContextWithUser) ~
-                  adminRoutes(samUser, samRequestContextWithUser) ~
-                  extensionRoutes(samUser, samRequestContextWithUser) ~
-                  groupRoutes(samUser, samRequestContextWithUser) ~
-                  userRoutesV1(samUser, samRequestContextWithUser) ~
-                  azureRoutes(samUser, samRequestContextWithUser)
-              }
+              publicTermsOfServiceRoutes ~
+                withActiveUser(samRequestContext) { samUser =>
+                  isWorkbenchAdmin(samUser) { isAdmin =>
```

Comment:
We should find another way of doing this. As implemented here, we're checking _every single call_ to see if the user is an admin, even if the route never needs admin permissions. It will introduce an API call to Google for every single request to Sam. Checking for admin permissions should only happen for routes that support admin functionality, to reduce overhead.